docs/en/kubernetes/Metrics-On-Kubernetes.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: global
+title: Metrics System On Kubernetes
+nickname: Metrics
+group: Kubernetes
+priority: 2
+---
+
+This documentation focus on how to get metrics from Alluxio deployed on Kubernetes.
+Refer to [Metrics-System]({{ '/en/operation/Metrics-System.html' | relativize_url }}) for general information
+about the Alluxio metrics.
+Refer to [Metrics-List]({{ '/en/reference/Metrics-List.html' | relativize_url }}) for the complete set of metrics.
+
+* Table of Contents
+{:toc}
+
+## Metrics Sink Configuration
```

Comment:
and the structure is all the things are under metrics sync configuration
but we are focus on how to get metrics on kubernetes instead of the configuration

Reply:
Or actually is about how to configure and how to get metrics on kubernetes

Reply:
Looks like all docs start from ##, mostly because the titles are inherently #?

There're how to get metrics in individual sections, so everything under `metrics sink configuration` is probably not a good idea. Thanks